examples/web-instrumentation/README.md

```diff
@@ -0,0 +1,384 @@
+  cloudbuild.googleapis.com \
+  cloudtrace.googleapis.com \
+  compute.googleapis.com \
+  container.googleapis.com \
+  containerregistry.googleapis.com \
+  logging.googleapis.com \
+  monitoring.googleapis.com
+```
+
+
+Install the JavaScript packages required by both the server and the browser:
+
+```shell
+npm install
+```
+
+## OpenTelemetry collector
+
+Open up a new shell to clone the OpenTelemetry collector contrib project, which
+contains the Stackdriver exporter
+
+```shell
+git clone https://github.com/open-telemetry/opentelemetry-collector-contrib
+cd opentelemetry-collector-contrib
+```
+
+Change directories and make
+
+```shell
+make otelcontribcol
+```
+
+Build the container
+
+```shell
+make docker-otelcontribcol
+```
+
+Tag it for Google Container Registry (GCR)
+
+```shell
+docker tag otelcontribcol gcr.io/$GOOGLE_CLOUD_PROJECT/otelcontribcol:latest
+```
+
+Push to GCR
+
+```shell
+docker push gcr.io/$GOOGLE_CLOUD_PROJECT/otelcontribcol
+```
+
+### Run the OpenTelemetry collector locally
+
+If you are running on GKE only, you do do not need to do this step.
+For running locally, the OpenTelemetry collector needs permissions and
+credentials to write to Stackdriver.
+
+Obtain user access credentials and store them for Application Default
+Credentials
+
+```shell
+gcloud auth application-default login \
+  --scopes="https://www.googleapis.com/auth/trace.append"
+```
+
+Install Go and run **This bit is not working for OT**
+
+```shell
+make otelcontribcol
+bin/linux/otelcontribcol --config=$WI_HOME/conf/otservice-config.yaml
+```
+
+## Browser code
+
+The browser code refers to ES2015 modules that need to be transpiled and bundled
+with the help of webpack. Make sure that the variable `agentURL` in
+`src\index.js` refers to localhost if running the OpenCensus agent locally or to
+the external IP of the OpenCensus agent if running on Kubernetes.
+
+In the original terminal, change to the browser code directory
+
+```shell
+cd browser
+```
+
+Install the browser dependencies
+
+```shell
+npm install
+```
+
+Compile the code
+
+```shell
+npm run build
+```
+
+## Run app locally
+
+The app can be deployed locally. First change to the top level directory
+
+```shell
+cd ..
+```
+
+To run the app locally type
+
+```shell
+node ./src/app.js
+```
+
+Open your browser at http://localhost:8080
+
+Fill in the test form to generate some load. You should see logs from both the
+Node.js server and the browser code in the console. You should see traces in
+Stackdriver.
+
+## Deploy to Kubernetes
+
+Create a cluster with 1 node and cluster autoscaling enabled
+
+```shell
+ZONE=us-central1-a
+NAME=web-instrumentation
+CHANNEL=regular # choose rapid if you want to live on the edge
+gcloud beta container clusters create $NAME \
+   --num-nodes 1 \
+   --enable-autoscaling --min-nodes 1 --max-nodes 4 \
+   --enable-basic-auth \
+   --issue-client-certificate \
+   --release-channel $CHANNEL \
+   --enable-stackdriver-kubernetes \
```

Comment:
This looks duplicated with the flag at the end. Remove this line?

Reply:
Thanks. I have removed it.